### PR TITLE
calculated TVL amount

### PR DESCRIPTION
**Pull request by josephchimebuka**

### CODE REVIEW

File: docker-compose.dev.yaml [hunk 1]
```diff
@@ -1,5 +1,3 @@
-version: '3.8'
```
Comment (djeck1432) [inline docker-compose.dev.yaml:1]: Why did you remove it?

File: web_app/db/crud.py [hunk 2]
```diff
@@ -362,23 +362,33 @@ def open_position(self, position_id: uuid.UUID, current_prices: dict) -> str | N
...
+                    db.query(Position)
                     .filter(Position.status == Status.OPENED.value)
-                    .scalar()
+                    .all()
                 )
-                return total_opened_amount
+                current_prices = await self.get_current_prices()
+
+                total_amounts = {}
+                for position in open_positions:
```
Comment (djeck1432) [inline web_app/db/crud.py:381]: we want to have as a result only one value, why we have separated total_amount per token?

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, fix pylint and tests  @josephchimebuka

**Comment by djeck1432** [comment]
How to run and find all issues with pylint
```
 pylint $(git ls-files '*.py') --disable=all --enable=C0114,C0115,C0116,C0301
 ```

File: docker-compose.dev.yaml (hunk 1 above)
Comment (josephchimebuka) [reply docker-compose.dev.yaml:1]: When I ran the docker build command it recommended I remove it before I can run it. I'll add it back and fix other issues ASAP

Comment (djeck1432) [reply docker-compose.dev.yaml:1]: hmmm...
send me this error pleae, it's really weird

File: web_app/db/crud.py (hunk 2 above)
Comment (djeck1432) [reply web_app/db/crud.py:381]: so it should be total of all token_amounts in USDC

**Comment by djeck1432** [comment]
@josephchimebuka Please, fix comments as soon as possible

**Comment by djeck1432** [comment]
This is duplicated PR
